pt/esbocos/originais/sumarizacao-narrativas-web-python.md

```diff
@@ -274,55 +253,55 @@ for item in contentsJSon["response_items"]:
     print(time)
 ```  
 
-# Conta-me Histórias
+# *Conta-me Histórias*
 
-O projeto Conta-me Histórias é um projeto desenvolvido por investigadores do Laboratório de Inteligência Artificial e Apoio a Decisão ([LIAAD](https://www.inesctec.pt/en/centres/liaad) — [INESCTEC](https://www.inesctec.pt/en/)) e afiliados às instituições [Instituto Politécnico de Tomar](https://www.ipt.pt) — [Centro de Investigação em Cidades Inteligentes (CI2)](http://www.ci2.ipt.pt/pt/); [Universidade do Porto](https://www.up.pt) e [Universidade de Innsbruck](https://www.uibk.ac.at/index.html.en). O projeto visa oferecer aos utilizadores a possibilidade de revisitarem tópicos do passado através de uma interface semelhante ao Google, que, dada uma pesquisa, devolve uma sumarização temporal das notícias mais relevantes preservadas pelo Arquivo.pt acerca desse tópico. Um vídeo promocional do projeto pode ser visualizado [neste vídeo](https://www.youtube.com/embed/fcPOsBCwyu8).
+O projeto *Conta-me Histórias* é desenvolvido por pesquisadores do Laboratório de Inteligência Artificial e Apoio a Decisão ([LIAAD](https://www.inesctec.pt/pt/centros/liaad)) — [INESCTEC](https://www.inesctec.pt/pt)) e afiliados às instituições [Instituto Politécnico de Tomar](https://www.ipt.pt) — [Centro de Investigação em Cidades Inteligentes (CI2)](http://www.ci2.ipt.pt/pt/), [Universidade do Porto](https://www.up.pt) e [Universidade de Innsbruck](https://www.uibk.ac.at/index.html.en) (em inglês). O projeto visa oferecer aos usuários a possibilidade de revisitarem tópicos do passado através de uma interface semelhante ao Google que, dada uma pesquisa, devolve uma sumarização temporal das notícias mais relevantes preservadas pelo Arquivo.pt acerca desse tópico. Um vídeo promocional do projeto pode ser visualizado [aqui](https://www.youtube.com/embed/fcPOsBCwyu8).

```

Comment:
O link no final da frase não está a abrir (https://www.youtube.com/embed/fcPOsBCwyu8).

Reply:
Hello @joanacvp,

In this case, I think the correct link would be: https://youtu.be/fcPOsBCwyu8

Reply:
Thank you very much @anisa-hawes. I already changed it :)